src/routes/User.jsx

```diff
@@ -0,0 +1,91 @@
+// React Imputs
+import React, { useContext, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+// Inrupt Imports
+import { useSession } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import Button from '@mui/material/Button';
+import Container from '@mui/material/Container';
+// Custom Hook Imports
+import { useStatusNotification } from '../hooks';
+// Utility Imports
+import { runCheckFiles, runNotification } from '../utils';
+// Context Imports
+import { SignedInUserContext } from '../contexts';
+// Component Imports
+import { ShowDocumentsModal } from '../components/DocumentModals';
+import FormSection from '../components/Form/FormSection';
+
+/**
+ * User - Page that displays the User's stored Documents
+ *
+ * @memberof Pages
+ * @name User
+ * @returns {React.ReactElement} The user page
+ */
+
+const User = () => {
```

Comment:
Is this needed? I feel like it displays the same information already present in clients and forms. Either way, I feel like it should have a different name. As is, it's confusing for both the developer and the case manager. The case manager won't know who the 'user' is, and developers may confuse this page with the `User` model object (which is supposed to refer to a generic user).

Maybe we can pass this idea through UX a bit more and revisit it in a later PR.

Reply:
I'm not sure what would be a better word for the user's own Pod storage. That's what the page would be for eventually while we move some functionalities over.

I'm thinking along the line of 'Personal Pod' or 'Own Pod', but 'User' is the best I could think of at the moment. I'm up for suggestions.

Reply:
'Personal Pod' is more apt in my opinion. Maybe 'Your Pod', 'Your Info', or something along those lines.

Reply:
Will consider this when we make the "User" page again; Will temporarily remove this for now

Reply:
> I'm not sure what would be a better word for the user's own Pod storage. That's what the page would be for eventually while we move some functionalities over.

Can you explain what you mean by this? Is the user's own Pod storage anything more than documents?

Reply:
"User" would eventually include upload, delete, fetch, and set permissions to documents. Each user of PASS would be able to access it there, and it'll only be related to their own Pod.

For now, I'm thinking of just leaving the cross Pod functionalities to "Clients" so that only case workers could work on other people's Pods.

Reply:
Am considering moving permission setting features to if we create a "Connections" page instead of in "User"/"My Documents" as a possible direction, see https://github.com/codeforpdx/PASS/pull/254#issuecomment-1597524146